src/pages/Main/Main.js

```diff
@@ -1,9 +1,73 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Item from '../Shopping/Item/Item';
+import './Main.scss';
+
+export default class Main extends Component {
+  constructor(props) {
+    super(props);
+    this.slideContainer = React.createRef();
+  }
+
+  state = {
+    slideIndex: 0,
+  };
+
+  slideNext() {
+    this.setState({ slideIndex: this.state.slideIndex + 1 }, () => {
+      this.slideContainer.current.style.transition = '  transform 2s';
+      this.slideContainer.current.style.transform = `translateX(-${
+        1184 * this.state.slideIndex
+      }px)`;
+    });
+  }
+
+  slidePrevious() {
+    this.setState({ slideIndex: this.state.slideIndex - 1 }, () => {
+      this.slideContainer.current.style.transition = ' transform 1s';
+      this.slideContainer.current.style.transform = `translateX(-${
+        1184 * this.state.slideIndex
+      }px)`;
+    });
+  }
 
-class Main extends React.Component {
   render() {
-    return <h1>d</h1>;
+    return (
+      <div className="mainPage">
+        <div className="slideOverFlow">
+          <div className="slideContentContainer" ref={this.slideContainer}>
+            <div className="eventSlide red"> </div>
+            <div className="eventSlide blue"> </div>
+            <div className="eventSlide yellow"> </div>
+            <div className="eventSlide gr"> </div>
+            <div className="eventSlide wh"> </div>
+            <div className="eventSlide bk"> </div>
+          </div>
+        </div>
+        <div className="slideIndexBar">
+          <div className="slideBtn" onClick={() => this.slidePrevious()}>
```

Comment:
- 화살표 함수로 바꿔주시면 this 가 고정되어서 문제가 사라집니다!
```suggestion
          <div className="slideBtn" onClick={this.slidePrevious}>
```